README.md

```diff
@@ -16,13 +16,38 @@ Let's get your machine setup to startup the application!
 ⚠️  We assume you already have ruby installed with your preferred version manager. This codebase supports [rbenv](
 https://github.com/rbenv/rbenv) and [asdf](https://github.com/asdf-vm/asdf-ruby).
 
+## 🤝 Contributing Guidelines 
+
+Please feel free to contribute! Priority will be given to pull requests that address outstanding issues and have appropriate test coverage. Focus on issues tagged with the next milestone for higher priority.
+
+To contribute:
+* Identify an unassigned issue
+* Assign the issue to yourself to avoid duplicated efforts (or request assignment by adding a comment)
+* Fork the repo if you're not a contributor yet
+* Create a new branch for the issue using the format `XXX-brief-description-of-feature`, where `XXX` is the issue number
+* Make sure tests run green locally and remotely. Create tests to validate your work fixes the issue(most of the time)
```

Comment:
I'd put this bullet directly under the 'fork the repo...' bullet and reword it something like - get the application up and running locally so that it works in the browser and when you run the test suite locally it passes.

We can then add bullets following the 'Create a new branch' bullet: 
- Make code changes related to the issue you are working on
- Comitt locally using descriptive...
- Add tests related to your work (most of the time)
- Ensure all tests pass successfully...

Let me know if unclear :)